stdlib/xml/dom/minidom.pyi

```diff
@@ -87,31 +87,31 @@ class NamedNodeMap:
     def keys(self): ...
     def keysNS(self): ...
     def values(self): ...
-    def get(self, name, value: Any | None = ...): ...
+    def get(self, name: str, value: Any | None = ...): ...
     def __len__(self) -> int: ...
     def __eq__(self, other: object) -> bool: ...
-    def __ge__(self, other: Any) -> bool: ...
-    def __gt__(self, other: Any) -> bool: ...
-    def __le__(self, other: Any) -> bool: ...
-    def __lt__(self, other: Any) -> bool: ...
-    def __getitem__(self, attname_or_tuple): ...
-    def __setitem__(self, attname, value) -> None: ...
-    def getNamedItem(self, name): ...
-    def getNamedItemNS(self, namespaceURI: str, localName): ...
-    def removeNamedItem(self, name): ...
-    def removeNamedItemNS(self, namespaceURI: str, localName): ...
-    def setNamedItem(self, node): ...
-    def setNamedItemNS(self, node): ...
-    def __delitem__(self, attname_or_tuple) -> None: ...
+    def __ge__(self, other: object) -> bool: ...
+    def __gt__(self, other: object) -> bool: ...
+    def __le__(self, other: object) -> bool: ...
+    def __lt__(self, other: object) -> bool: ...
```

Comment:
This smells a bit off to me. It's true that you _can_ write bizarre code like this, and no exception will be raised at runtime:

```pycon
>>> from xml.dom.minidom import NamedNodeMap
>>> NamedNodeMap('foo', 'bar', 'baz') < object()
False
```

But... that doesn't mean that you necessarily _should_. Code like that just doesn't make much sense.

I feel like `NamedNodeMap`s are _probably_ only meant to be compared to other `NamedNodeMap`s?

Reply:
I assumed that, but when I looked at https://docs.python.org/3/library/functions.html#id it mentioned that `id` I saw the CPython implementation detail that it's returning the memory location. As a result, I wasn't sure if it was intended to rely on the implementation details of `id` as a shortcut to inferring details about parsing progression. I'll restrict it though as requested